cgotorch/cgotorch.cc

```diff
@@ -16,10 +16,18 @@ Tensor RandN(int rows, int cols, int require_grad) {
   return new at::Tensor(t);
 }
 
-Tensor MM(Tensor a, Tensor b) {
-  at::Tensor c =
-      at::mm(*static_cast<at::Tensor *>(a), *static_cast<at::Tensor *>(b));
-  return new at::Tensor(c);
+char *MM(Tensor a, Tensor b, Tensor *result) {
+  try {
+    at::Tensor c =
+        at::mm(*static_cast<at::Tensor *>(a), *static_cast<at::Tensor *>(b));
+    *result = new at::Tensor(c);
+    return nullptr;
+  } catch (const std::exception& e) {
+    auto len = strlen(e.what());
+    auto r = new char[len + 1];
```

Comment:
So every time we call MM, we need to call FreeString() to release the error right?  I don't see the call to FreeString in the examples?